endpoints/events/event.go

```diff
@@ -162,6 +166,10 @@ func ParseEventRequest(r *http.Request) (*analytics.EventRequest, []error) {
 		errs = append(errs, err)
 	}
 
+	if err := readIntegrationType(event, r); err != nil {
+		errs = append(errs, err)
```

Comment:
Can you add coverage for this line? You should be able to just add another test which is basically a copy of `TestShouldReturnBadRequestWhenFormatValueIsInvalid`.